packages/kit/src/core/adapt/prerender.js

```diff
@@ -26,6 +26,12 @@ function get_src(attrs) {
 	return match && (match[1] || match[2] || match[3]);
 }
 
+/** @param {string} attrs */
+function get_rel(attrs) {
```

Comment:
Can't we just call `element.getAttribute('rel')`?

Reply:
No, because we have `a` as a string, not a node